lesson_23_selenide/src/test/java/pageobject_model/HeaderMenu.java

```diff
@@ -0,0 +1,127 @@
+package pageobject_model;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Configuration;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.CollectionCondition.size;
+import static com.codeborne.selenide.Condition.attribute;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+
+
+public class HeaderMenu {
+    static By profileButton = By.id("fm-account-dropdown");
+    static By loginModal = By.className("modal-content");
+    static By emailField = By.id("emailLoginInput");
+    static By passwordField = By.id("passwordLoginInput");
+    static By loginButton = By.id("popup-login-button");
+    static By alertToast = By.className("alert-block");
+    static By alertToastText = By.className("fm-alert-text");
```

Comment:
if you use field only once for some specific scenario - you could declare this field in the method